scripts/vagrant/init_environment.sh

```diff
@@ -98,33 +108,54 @@ if [ ! -d $TWWEB_TASKD_DATA ]; then
     if [ -z "$TRAVIS" ]; then
         service taskd start
     fi
-fi
 
-which task
-if [ $? -ne 0 ]; then
-    cd $TWWEB_TASKD_DATA/src
-    wget http://taskwarrior.org/download/task-2.3.0.tar.gz
-    tar xzf task-2.3.0.tar.gz
-    cd task-2.3.0
-    cmake .
-    make
-    checkinstall --default
-    cp /var/taskd/src/task-2.3.0/task_2.3.0-1*.deb /tmp
+    set +e
+    which task; RETVAL=$?
+    set -e
+    if [ $RETVAL -ne 0 ]; then
+        TASK_VERSION="task-2.3.0"
+        echo "installing $TASK_VERSION"
+        cd $TWWEB_TASKD_DATA/src
+        wget -nv http://taskwarrior.org/download/$TASK_VERSION.tar.gz
+        tar xzf $TASK_VERSION.tar.gz
+        cd $TASK_VERSION
+        cmake .
+        make
+        checkinstall --default
+        cp /var/taskd/src/$TASK_VERSION/*.deb /tmp
+    fi
 fi
 
-cd $MAIN_DIR
-npm install -g ember-cli@0.1.7 bower@1.3.12
-ember install
+# copy MAIN_DIR into a temp folder and run npm there
+# to avoid EPERM errors on NFS shared folders in vagrant
+# See: https://github.com/npm/npm/issues/3565
+cd $STARTING_DIR
+cp -a $MAIN_DIR /tmp/twweb
+chown -R vagrant. /tmp/twweb
+cd /tmp/twweb
+
+echo "installing ember-cli and bower"
+npm install -g ember-cli bower
```

Comment:
I feel really embarrassed to see that these versions are still correct, npm ls on a successfull build reports those same versions.

I think running the vagrant share on nfs or running ember install before npm install was causing npm package problems from the start.